tests/personio/test_personio_source.py

```diff
@@ -0,0 +1,35 @@
+import dlt
+import pytest
+
+from sources.personio import personio_source
+from tests.utils import ALL_DESTINATIONS, assert_load_info, load_table_counts
+
+
+@pytest.mark.parametrize("destination_name", ALL_DESTINATIONS)
+def test_all_resources(destination_name: str) -> None:
+    pipeline = dlt.pipeline(
+        pipeline_name="personio",
+        destination=destination_name,
+        dataset_name="shopify_data",
```

Comment:
you can use names for all tests like "test_data" or "test_pipeline", not to rename them every time